packages/teleport/src/services/audit/makeEvent.ts

```diff
@@ -409,6 +409,11 @@ export const formatters: Formatters = {
       return `User [${user}] has ended interactive session [${event.sid}] on node [${node}] `;
     },
   },
+  [CodeEnum.SESSION_REJECT]: {
+    desc: 'Session Rejected',
+    format: ({ user, login, server_id, reason }) =>
+      `User [${user}] was denied session for [${login}@${server_id}] because [${reason}]`,
```

Comment:
`was denied session for` -> `was denied access to`

Reply:
revised